### PR TITLE
fix(dashboard): refactor board-list & some style changes

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

- Refactor board-list
- Edit board's contents
- Style changes

### CODE REVIEW

File: src/services/dashboards/all-dashboards/AllDashboardsPage.vue [hunk 1]
```diff
@@ -30,13 +30,13 @@
              class="dashboard-list-wrapper"
         >
             <dashboard-board-list v-if="scopeStatus !== SCOPE_TYPE.PROJECT && workspaceDashboardList.length > 0"
```
Comment (yuda110) [inline src/services/dashboards/all-dashboards/AllDashboardsPage.vue:32]: question: Can't we use `DASHBOARD_SCOPE ` here too?

**Review by song2kim — APPROVED** [review]
LGTM!!!

File: src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue [hunk 2]
```diff
@@ -136,16 +140,26 @@ export default defineComponent<DashboardBoardListProps>({
     setup(props) {
         const state = reactive({
             thisPage: 1,
-            dashboardScopeType: computed(() => DASHBOARD_SCOPE[props.scopeType]),
-            dashboardScopeTypeForView: computed(() => (props.scopeType === DASHBOARD_SCOPE.DOMAIN ? 'Workspace' : 'Project')),
+            dashboardScopeKey: computed(() => (props.scopeType === DASHBOARD_SCOPE.DOMAIN ? DOMAIN_SCOPE_KEY : PROJECT_SCOPE_KEY)),
+            projectItems: computed(() => store.state.reference.project.items),
```
Comment (yuda110) [inline src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue:144]: suggestion: Please use getters except when neccessary! ([Why should we use getters?](https://www.notion.so/spaceone-io/Reference-Store-getters-key-10ce040c25894624a62ad05d51c1d1cc))

```suggestion
            projects: computed<ProjectReferenceMap>(() => store.getters['reference/projectItems']),
```

**Review by yuda110 — APPROVED** [review]
LGTM!!! I left some comments!

File: src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue (hunk 2 above)
Comment (piggggggggy) [reply src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue:144]: Thanks!!

**Pull request merged**